Job.tscn

```diff
@@ -0,0 +1,14 @@
+[gd_scene load_steps=3 format=3 uid="uid://bii6mxi26g43c"]
```

Comment:
Is this file being used?

Reply:
Might've stayed during the merge/rebase, I'll delete it